docker-run.sh

```diff
@@ -23,6 +26,9 @@ WORKDIR="/projects/src/${GO_MODULE}"
 # Container image
 IMAGE_NAME="quay.io/devfile/kubernetes-api-build-prerequisites:latest"
 
+# Run script to set env var for podman if necessary
+. ./setenv.sh
+
```

Comment:
I'm not sure I like hiding the alias in a separate script and controlled by an env var -- it's hiding a lot of functionality here (if you don't notice these two lines at the top it looks like it just uses docker).